dune-project

```diff
@@ -1 +1,2 @@
 (lang dune 1.6)
+(name bigstringaf)
```

Comment:
What does this do?

Reply:
Hmm, honestly I don't remember. I suspect some version of dune asked for it, but I can't reproduce this anymore.
```suggestion
```